### DISCUSSION TITLE
"Guiding Flashlight" Mod Idea

### DISCUSSION THREAD

**Discussion by EM5I3R3S** [ppy/osu#26570] [Ideas]
Hi everyone 👋

I came up with idea of a mod, that can (in theory) **significantly** simplify the process of memorizing a map, in order to set neat scores with FL! This can be a real thing!

🔦  **Introducing the Concept:**  

Imagine the visible area of Flashlight moving automatically to the next notes, without player interaction, just like Auto mod playing with Flashlight. And here's the main idea: the player should play the map, while this flashlight guiding the player where to look next! This approach can make learning a map almost a cakewalk! 

The main advantage of this modification is that when playing with it, the process of memorizing a song is greatly simplified, since the visible area of the flashlight moves automatically to the next notes, regardless of where the player’s cursor position. 

Comparing the original Flashlight and this new mod, it is obvious that the new mod is much easier to play with, but I want this modification to be introduced so that players can memorize maps in a much easier way, so that the players can set epic cool score on maps!

**Comment by nana123126** [comment]
Using FL+AP in the Skin layout editor has a similar effect.
And I think this only needs to be added to the mod customization, not a new mod. (like HD)
